data_submission/src/encipher/types.rs

```diff
@@ -0,0 +1,29 @@
+use alloy_primitives::{Address, Signature};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct EncryptRequest {
+    pub app_id: u32,
```

Comment:
as an additional param take turbo da app id also here. Reason being inside tee the user's app id should be use to map it to the respective MPC